lib/commands/changelog.js

```diff
@@ -11,9 +13,23 @@ module.exports = {
 
   availableOptions: [],
 
-  anonymousOptions: [],
+  anonymousOptions: [
+    '<from-rev>',
+    '<release-name>',
+  ],
```

Comment:
It looks like you intend to invoke `ember-cli-changelog` from the command line.  You can actually do this easier via good old `require`.

```
try {
  var generateChangelog = require('ember-cli-changelog/lib/tasks/release-with-changelog');
} catch (e) {
  throw new Error("Please ensure that you have installed ember-cli-changelog");
}
```